### ISSUE TITLE
Low Disk Space : increase default limit

### ISSUE THREAD

**Issue by tomdereub** [nextcloud/desktop#3398]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Expected behaviour
I don't want nextcloud to be able to saturate my disk up to crash my computer.

### Actual behaviour
As it's described here : https://github.com/nextcloud/desktop/issues/487 and here : https://docs.nextcloud.com/desktop/2.3/advancedusage.html#low-disk-space,
nextcloud will stop syncing if free disk space falls below 50 MB. I find it's far to low, with just 50 MB of free space a windows will stall quickly (just try to open a big file in any program, it doesn't manage to create the temporary files it needs).

So I suggest to increase the default value of OWNCLOUD_CRITICAL_FREE_SPACE_BYTES and OWNCLOUD_FREE_SPACE_BYTES to something like 1 GB, I think it would be more secure.

### Client configuration
Client version: 3.2.2
Operating system: Windows 10
OS language: French

### Server configuration
Nextcloud version: 20.0.7

**Comment by tomdereub** [comment]
Is it possible to open this issue again ? It seems to me that it's really easy to solve, just change the default value in this file : https://github.com/nextcloud/desktop/blob/b7f67c0adf3d3fd7f99673d7622ba86ce960838c/src/libsync/owncloudpropagator.cpp
Lines 53 and 66.
I'm sorry I'm not able to make a PR.

**Comment by joshtrichards** [comment]
I agree 50 MB is likely relatively ubiquitously too low. Any value we pick here is relatively arbitrary, but perhaps we bump:

- OWNCLOUD_CRITICAL_FREE_SPACE_BYTES to 512MB
- OWNCLOUD_FREE_SPACE_BYTES to 1GB

Two spots in the docs will need to be updated as well:

https://github.com/search?q=repo%3Anextcloud%2Fdesktop%20OWNCLOUD_CRITICAL_FREE_SPACE_BYTES&type=code

**Comment by cryptogopher** [comment]
1. If normal Nexcloud client operation makes someone's Windows stall, then it's not Nextcloud's problem, but one of environment configuration. One should just not sync to the OS file system, but to a separate file system that does not create such risk.

2. I rememeber that at some point it was possible to set the minium amount of free space for Nextcloud client to be able to operate - as a configuration option. Now I cannot see this option anymore. What happened to it?

3. As can be seen, setting free disk space limits through environment variables is too hard task for an average user. It results in setting some arbitrary values in the source code, affecting everyone, e.g. me, where after last upgrade Nextcloud stopped syncing to the dedicated filesystem having almost 1GB free space (which is more than enough for my setup). Shouldn't it be possible to set the value in some easier manner, as before (sse p. 2)? I'm a Linux user for ~20 years and I've already lost more time than I would like to while searching for how to set environment variable for an autostart task on XFCE. It turns out it will be easier for me to just patch the source code, as I'm on Gentoo and can use epatch system, but god, have mercy for the other poor people out there...
